GitUpKit/Views/GIMapViewController+Operations.m

```diff
@@ -803,8 +804,10 @@ - (void)_pushLocalBranch:(GCHistoryLocalBranch*)branch toRemote:(GCRemote*)remot
             NSError* localError;
             GCHistoryLocalBranch* updatedBranch = [self.repository.history historyLocalBranchForLocalBranch:branch];  // Reload branch to check upstream!
             GCRemoteBranch* remoteBranch = [self.repository findRemoteBranchWithName:[NSString stringWithFormat:@"%@/%@", remote.name, branch.name] error:&localError];
+            BOOL askSetUpstreamOnPushSetting = [[NSUserDefaults standardUserDefaults] boolForKey:kUserDefaultsKey_AskSetUpstreamOnPush];
```

Comment:
Let's also call this simply `askSetUpstreamOnPush` as that is more of a boolean statement.